### PR TITLE
Convert into a "real" polyfill

### PR DESCRIPTION
**Pull request by mansona**
Fixes https://github.com/kellyselden/ember-array-helper/issues/89

### CODE REVIEW

**Comment by kellyselden** [comment]
Thanks for picking this up.

**Comment by mansona** [comment]
No worries 😂 also don't thank me until it's done 🙃 😂 

I opened this early just to see if we would get a failing CI for ember beta and canary here too 👍

**Comment by mansona** [comment]
@kellyselden this is ready for review now 👍 I'm not too happy that I added the change to remove `objectAt()` in this same PR but it would fail CI if it wasn't in the same PR 🤷‍♂ 

I guess it's not such a big deal as it's only changing the dummy app 😂

File: tests/dummy/app/components/my-component.js [hunk 1]
```diff
@@ -3,6 +3,6 @@ import Component from '@ember/component';
 
 export default Component.extend({
   secondItem: computed('array.[]', function() {
-    return this.get('array').objectAt(1);
```
Comment (kellyselden) [inline tests/dummy/app/components/my-component.js]: Hmm I wonder why I used this in the first place.

**Comment by kellyselden** [comment]
Released as v5.1.0

**Pull request merged**